Core/clim-core/presentations/translators.lisp

```diff
@@ -293,7 +293,7 @@ and used to ensure that presentation-translators-caches are up to date.")
   (let* ((command-table (find-command-table command-table))
          (from-name     (presentation-type-name from-type))
          (to-name       (presentation-type-name to-type))
-         (cache-key     (cons from-type to-name))
+         (cache-key     (cons from-name to-name))
```

Comment:
commit message: the part about "or types colliding" is not adequate anymore

Reply:
I think it is still true for this commit because the `stupid-subtypep` change is in the next commit. Should I mention that the situation will change with the next commit?

Reply:
there is no need.